src/apps/answers/service.py

```diff
@@ -1075,11 +1087,17 @@ async def get_summary_activity_flows(
             submitted_activity_flows[flow_id] = max(submit_date, date) if date else submit_date
             submitted_activity_flows[version_id] = submit_date
 
+        flow_histories = await flow_crud.get_by_applet_id_version(current_applet_version)
```

Comment:
Last flow version store in `flows` table. You can reuse FlowsCRUD(self.session).get_by_applet_id(applet_id)